docs/developer-guide/governance.md

```diff
@@ -55,9 +55,9 @@ Reviewers have additional responsibilities over and above those of a Committer.
 Reviewers fulfill all requirements of Committers, and also:
 
 * Contributions can be reviewed by other Reviewers, but this is not explicitly required.
-* Are not required to submit pull requests for changes, however, they are strongly encouraged to do so when the change is non-trivial.
-* May merge their own pull requests once they have collected the feedback they deem necessary.
-* Can label issues as they are submitted (but must not add the "accepted" label).
+* May merge external pull requests for accepted issues upon reviewing and approving the changes.
+* May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one Committer/Reviewer comment stating they've looked at the code.)
```

Comment:
Do you think it would be OK, to specify that that applies mostly to ESLint repository itself? (I merge my changes into website quite a bit without feedback, cause nobody looks at website repo:-) And also, maybe a clause to say that if the change is blocking/urgent it can be merged without review?